src/transformers/models/jukebox/modeling_jukebox.py

```diff
@@ -2281,47 +2262,21 @@ class JukeboxPreTrainedModel(PreTrainedModel):
 
     def _init_weights(self, module):
         std = self.config.init_std
-        if isinstance(module, nn.Linear):
-            module.weight.data.normal_(mean=0.0, std=std)
-            if module.bias is not None:
-                module.bias.data.zero_()
-        elif isinstance(module, nn.Embedding):
-            module.weight.data.normal_(mean=0.0, std=std)
-
-        # TODO handle all the initialisation and zero_out from the rest of the code
+        init_scale = self.config.init_scale
+
+        if isinstance(module, nn.Embedding): # embed_tokens
+            module.weight.data.normal_(mean=0.0, std= 0.02 * init_scale)
+        elif isinstance(module,nn.Parameter):
+            module.pos_emb.data.normal_(mean=0.0, std=0.01 * init_scale)
```

Comment:
The function is recursively applied, so with this, every weight will be initialized like this and all the rest of the tests is not usefuly.